data/json/items/tool/firefighting.json

```diff
@@ -192,7 +192,7 @@
     "techniques": [ "WBLOCK_1", "SWEEP" ],
     "flags": [ "DURABLE_MELEE", "REACH_ATTACK", "NONCONDUCTIVE", "SHEATH_SPEAR", "ALWAYS_TWOHAND" ],
     "weight": "3100 g",
-    "volume": "3250 ml",
+    "volume": "1250 ml",
```

Comment:
```suggestion
    "//": " If the pole itself has a diameter of around 3 cm which is thick, to be frank, then we half it for the radius -> 1,5 cm, square it -> 2,25, mulitplicate with pi -> ~7,07 cm² multiplicate with 160 cm lengh -> 1130 cm³ which is 1,13 liters. We add the hook which is around 100 ml of volume."
    "volume": "1250 ml",
```

Reply:
thanks that my comment is "json comment" worthy.